editoast/src/models/projects.rs

```diff
@@ -38,24 +38,35 @@ pub struct Project {
     #[diesel(deserialize_as = String)]
     pub name: Option<String>,
     #[diesel(deserialize_as = String)]
+    #[derivative(Default(value = "Some(String::new())"))]
     pub description: Option<String>,
     #[diesel(deserialize_as = String)]
+    #[derivative(Default(value = "Some(String::new())"))]
     pub objectives: Option<String>,
     #[diesel(deserialize_as = String)]
+    #[derivative(Default(value = "Some(String::new())"))]
     pub funders: Option<String>,
     #[diesel(deserialize_as = i32)]
+    #[derivative(Default(value = "Some(0)"))]
     pub budget: Option<i32>,
     #[diesel(deserialize_as = Option<i64>)]
     #[diesel(column_name = "image_id")]
     pub image: Option<Option<i64>>,
     #[diesel(deserialize_as = NaiveDateTime)]
+    #[derivative(Default(value = "Some(Utc::now().naive_utc())"))]
```

Comment:
I think this will create a bug. By default, `creation_date` will now be the current date/time. However, in the  `patch` method of `views/projects`, first line, `data.into_inner()`, it will put this default value for `creation_date`, instead of `null` as before, and hence will update it accordingly, resulting in the modification of the field `creation_date`.

Reply:
Nicely spotted :) I removed this Default value